Documentation/Debugger/gdb/Windows Subsystem for Linux.md

```diff
@@ -0,0 +1,88 @@
+# Windows 10's Windows Subsystem for Linux
+With the release of Windows 10 Creator's Update, you will now be able to use Visual Studio Code and the Microsoft C/C++ extension to debug your `Windows Subsystem for Linux (WSL)` Bash on Ubuntu projects.
+
+Code can be written on Windows itself using VSCode and debugged through `bash.exe` to the Bash on Windows layer. 
+
+**NOTE: Creator's Update is required due to bugfixes within the subsystem that we rely on to provide debugging. Debugging using a previous version of WSL is unsupported and likely will not work.**
+
+## Prerequisites
+* [Windows 10 Creator's Update with Windows Subsystem for Linux and Bash.](https://msdn.microsoft.com/en-us/commandline/wsl/install_guide)
+* Install g++/gcc and gdb to allow compiling and debugging.
```

Comment:
Added `WSL` to the sentence. The command you provided isn't correct but i'll add something in regards to using the package manager to install tools.